tokenomics-explainers/003-rework-foundation.md

```diff
@@ -64,30 +65,34 @@ The NO earns solo staking APY on their bond (0.334 ETH), 14% commission on the b
 
 The end result is that the Node Operator would make **1.14x solo staking APY right now**. At maturity, RPL would approach ~100% staked, at which point RPL rewards and Inflation to NOs cancel each other out resulting in **1.01x solo staking APY**. Back to the current state, if a Node Operator loses eligibility for RPL rewards (which often happens due to falling RPL/ETH price ratio), the loss of the 0.14ETH in RPL rewards would result in **0.82x solo staking APY**.
 
+TODO -- update to new no_share
 **Proposal using ETH-only allocation**  
 <img src="../assets/tokenomics-explainers/003-figure-3.png" alt="Figure 3" width="1000px"></br>
 _Figure 3 - Proposal ETH-only Sankey Diagram_
 
-The NO earns solo staking APY on their bond (0.334 ETH), and 3.5% commission on the borrowed ETH (0.035 ETH).
+The NO earns solo staking APY on their bond (0.334 ETH), and 5% commission on the borrowed ETH (0.05 ETH).
 
-The end result is that the Node Operator would make **1.11x solo staking APY right now**.
+The end result is that the Node Operator would make **1.15x solo staking APY right now**.
 
+TODO -- update to new no_share and voter_share
 **Proposal using current minimum LEB8 allocation**  
 <img src="../assets/tokenomics-explainers/003-figure-4.png" alt="Figure 4" width="1000px"></br>
 _Figure 4 - Proposal w/minimum LEB8 Allocation Sankey Diagram_
 
-The NO earns solo staking APY on their bond (0.334 ETH), 3.5% commission on the borrowed ETH (0.035 ETH), 5% revenue share on their vote-eligible RPL (0.05 ETH)[^2], and 5.5% revenue share on their RPL (0.033 ETH)[^3]. Meanwhile, they lose RPL value to inflation going to DAO expenses (0.036 ETH). 
+The NO earns solo staking APY on their bond (0.334 ETH), 5% commission on the borrowed ETH (0.05 ETH), and 9% revenue share on their vote-eligible RPL (0.09 ETH)[^2]. Meanwhile, they lose RPL value to inflation going to DAO expenses (0.036 ETH). 
 
-The end result is that the Node Operator would make **0.96x solo staking APY right now**. Note that voter share depends on your exposure to RPL relative to other NOs. Eg, if half of the validators are ETH-only and half look like this NO, then the voter share reward to this NO would double to 0.1 ETH; that would result in **1.07x solo staking APY**. This dynamic will be explored further in the Case Study below.
+The end result is that the Node Operator would make **1.09x solo staking APY right now**. Note that voter share depends on your exposure to RPL relative to other NOs. Eg, if half of the validators are ETH-only and half look like this NO, then the voter share reward to this NO would double to 0.1 ETH; that would result in **1.21x solo staking APY**. This dynamic will be explored further in the Case Study below.
```

Comment:
Hmm... I got 1.009x solo staking, not 1.09x?
Math:
solo = (8+0.334)/8 = **4.175% APY** (conditions laid out originally were 4.17%, but we rounded to 3 digits
RP = [(8+0.334+0.05) + (2.4+0.09) - 0.036]/10.4 = 10.838/10.4 = **4.21% APY**

4.21/4.175 = **1.009x solo staking APY**

Rounded to 2 digits = **1.01x**

Reply:
Also need to update:
> voter share reward to this NO would double to 0.18 ETH; that would result in **1.22x solo staking APY**

Math:
 RP = [(8+0.334+0.05) + (2.4+0.18) - 0.036]/10.4 = 10.928/10.4 = **5.077% APY**
 5.077/4.175 = 1.216 (or rounded to 2 digits = **1.22 solo staking APY**)